### ISSUE TITLE
Whole Script.

### ISSUE THREAD

**Issue by TurtMaster2000** [TurtMaster2000/Debugging-Script#1]
The entire thing has some sort of problem, it might have some sort of IDE problem, Java, Python (My least favorite), or C++ Etc. Thanks for trying.

**Comment by EXODUS10000** [comment]
HI this is cole your project is really cool i have know idea what it is????

**Comment by TurtMaster2000** [comment]
Well, Nice to know you care. I hope you can help me sometimes, with programming like this, you need to know what your doing. But it's not like I know what I'm doing anyway so I can't be talking.

**Comment by EXODUS10000** [comment]
yes i will try to i am not that good at all this stuff hopefully we can see each other again so you can teach me programing you are right i have no idea what i am doing

**Comment by EXODUS10000** [comment]
I rode the doom fall at universal

**Comment by TurtMaster2000** [comment]
Hi, this is completely public, just letting you know. I want you to be more careful about telling everyone where you went on vaca.

**Comment by EXODUS10000** [comment]
okay

**Comment by EXODUS10000** [comment]
got it
